core/templates/pages/exploration-editor-page/exploration-save-and-publish-buttons/exploration-save-and-publish-buttons.component.ts

```diff
@@ -66,7 +71,30 @@ angular.module('oppia').component('explorationSaveAndPublishButtons', {
         ExplorationSaveService.discardChanges();
       };
 
+      let isModalDisplayed = false;
+
       $scope.getChangeListLength = function() {
+        var countChanges = ChangeListService.getChangeList().length;
+
+        const minChangesDisplayPrompt = 50;
```

Comment:
Named constants generally use all caps + underscores. Can we do that here?

Reply:
DONE